### PR TITLE
Just started

### PR DESCRIPTION
**Pull request by Sumeh92**
Finished the project.

### CODE REVIEW

File: t04_refactored.py [hunk 1]
```diff
@@ -135,9 +139,37 @@ def team_5_adv():
 
 
 def team_6_adv():
+    """
+
```
Comment (sheggen) [inline t04_refactored.py:143]: Add the docstring information (what's this function do?)

File: t04_refactored.py [hunk 2]
```diff
@@ -135,9 +139,37 @@ def team_5_adv():
...
+    :return:
+    """
     pass
```
Comment (sheggen) [inline t04_refactored.py:146]: Remove `pass`, it is a placeholder for an empty function. Now that it's not empty, remove `pass`

File: t04_refactored.py (hunk 1 above)
Comment (Sumeh92) [reply t04_refactored.py]: I've fixed this.

File: t04_refactored.py (hunk 2 above)
Comment (Sumeh92) [reply t04_refactored.py]: I've fixed this.

File: t04_refactored.py [hunk 3]
```diff
@@ -135,9 +139,36 @@ def team_5_adv():
 
 
 def team_6_adv():
-    pass
-    # TODO Add your code here
+    """
+    This makes the adventure choose which path they would like to embark on.
```
Comment (sheggen) [inline t04_refactored.py:143]: Your Google doc link and names should be in the docstring.

**Pull request merged**